src/WinRT.Runtime/Projections/EventHandler.cs

```diff
@@ -323,13 +325,14 @@ static unsafe EventHandler()
             var nativeVftbl = ComWrappersSupport.AllocateVtableMemory(typeof(EventHandler), sizeof(global::WinRT.Interop.IDelegateVftbl));
             *(global::WinRT.Interop.IDelegateVftbl*)nativeVftbl = AbiToProjectionVftable;
             AbiToProjectionVftablePtr = nativeVftbl;
+            ComWrappersSupport.RegisterDelegateFactory(typeof(global::System.EventHandler), CreateRcw);
         }
 
 #if !NET
         public static global::System.Delegate AbiInvokeDelegate { get; }
 #endif
 
-        private static readonly Guid IID = new(0xc50898f6, 0xc536, 0x5f47, 0x85, 0x83, 0x8b, 0x2c, 0x24, 0x38, 0xa1, 0x3b);
+        public static Guid IID { get; } = global::WinRT.Interop.IID.IID_EventHandler;

```

Comment:
We don't need the duplicate storage here, we can probably optimize it:
```suggestion
        public static Guid IID => global::WinRT.Interop.IID.IID_EventHandler;

```